Fix issue related to service export (#183)
submariner-io/submariner-website: src/content/architecture/service-discovery/_index.en.md

The workflow is as follows:

- Lighthouse agent connects to the broker's K8s API server.

> - For every service in the local cluster for which an exported service is created, the agent creates a corresponding

Reviewer: ```suggestion
- For every service in the local cluster for which a ServiceExport has been created, the agent creates a corresponding
```
